test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests_CSharp.cs

```diff
@@ -40,6 +42,48 @@ public SamplesEndToEndTests_CSharp(TestFixture fixture)
             _settingsManager = ScriptSettingsManager.Instance;
         }
 
+        [Fact]
+        public async Task ExtensionWebHook_Succeeds()
```

Comment:
Apparently we didn't have any E2E test for the webhook endpoint in this solution